EzPC/EzPC/ast.ml

```diff
@@ -218,18 +230,30 @@ let rec typ_to_string (t:typ) :string =
        | UInt64 -> "uint64"
        | Int32  -> "int32"
        | Int64  -> "int64"
+       | Float  -> "float"
        | Bool -> "bool"
      in
      prefix ^ " " ^ bt_str
   | Array (quals, t, e) ->
      let qual_string = if quals |> List.mem Immutable then "const " else "" in
      qual_string ^ typ_to_string t ^ "[" ^ expr_to_string e ^ "]"
 
+let basetype_to_secfloat (t:base_type) :string =
+  match t with
+  | UInt32 | UInt64 | Int32 | Int64 -> "fix"
+  | Float -> "fp"
+  | Bool -> "bool"
+
+let basetype_to_secfloat_backend (t:base_type) :string = "__" ^ (basetype_to_secfloat t) ^ "_op"
```

Comment:
This (and the next one) also, move to codegen?

Reply:
moved.